components/Badge.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+    faCheckCircle,
+    faExclamationTriangle
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const badgeData = {
+    'verified': {
+        icon: faCheckCircle,
+        title: 'Verified',
+        className: "text-green-600 w-6",
+        textClassName: 'text-lg'
+    },
+    'unresponsive': {
+        icon: faExclamationTriangle,
+        title: 'Not Verified',
+        className: 'text-yellow-400 w-6',
+        textClassName: 'text-lg'
+    },
+    'unverified': {
+        icon: faExclamationTriangle,
+        title: 'Not Verified',
+        className: 'text-yellow-400 w-6',
+        textClassName: 'text-lg'
+    }
+};
+
+const Badge = ({
+    badgeType,
+}) => {
+    if (!badgeType || (badgeType && !badgeData[badgeType.toLowerCase()])) {
```

Comment:
Refer: https://life-bo2dzdlc7-coronasafe.vercel.app/maharashtra/nashik/medicine

Some of the Cards have no `verified` and `not verified` icon

So you gotta handle that case too. (Would Make sense right?)

Reply:
I would make the Changes :)

Reply:
I think I know how to handle this case, I just need to use || 'unverified'. I'll do it

Reply:
This is done

Reply:
Yeah looks good 🚀 Thanks for This!